uaclient/status.py

```diff
@@ -19,7 +58,6 @@ class TxtColor:
     ACTIVE: TxtColor.OKGREEN + ACTIVE + TxtColor.ENDC,
```

Comment:
could instead reference ApplicationStatus.ENABLED.value right?

Reply:
Yep, as above, this will come.